new_django_project/blog/api/models.py

```diff
@@ -9,3 +9,12 @@ class Post(models.Model):
     class Meta:
         ordering = ['created']
 
+class Comment(models.Model):
+    created = models.DateTimeField(auto_now_add=True)
+    body = models.TextField(blank=False)
+    owner = models.ForeignKey('auth.User', related_name='comments', on_delete=models.CASCADE)
+    post = models.ForeignKey('Post', related_name='comments', on_delete=models.CASCADE)
```

Comment:
[Just a DB modeling question] Why the decision of having the `comment` to reference the `post` and not the `post` to "own" the `comment`?